### PR TITLE
Add font and additional image mime types

### PR DESCRIPTION
**Pull request by sorcix**
Adds mimetypes for:

* `bmp` and `webp` images
* `otf`, `ttf`, `woff` and `woff2` fonts

### CODE REVIEW

File: src/mime/mod.rs [hunk 1]
```diff
@@ -53,11 +53,19 @@ impl Mime {
...
+            "css" => Some(CSS),
             "html" => Some(HTML),
+            "ico" => Some(ICO),
             "js" | "mjs" | "jsonp" => Some(JAVASCRIPT),
             "json" => Some(JSON),
-            "css" => Some(CSS),
+            "otf" => Some(OTF),
             "svg" => Some(SVG),
+            "ttf" => Some(TTF),
+            "txt" => Some(TXT),
```
[the remark this answers is not in the document]
Comment (sorcix) [reply src/mime/mod.rs]: You're right, fixed.

**Comment by sorcix** [comment]
As I had to fix the TXT/PLAIN issue and found out I also needed support for `wasm` files, I've added a few more mime types in a separate commit.

Not sure about the `SEVENZIP` constant. It doesn't work with `7ZIP` because of the number, any suggestions? :-)

**Comment by yoshuawuyts** [comment]
@sorcix it seems we're missing another `cargo fmt` pass on this; CI is failing

> Not sure about the `SEVENZIP` constant. It doesn't work with `7ZIP` because of the number, any suggestions? :-)

hah, that seems fine imo (:

**Comment by yoshuawuyts** [comment]
@sorcix we just merged https://github.com/http-rs/http-types/pull/385 which fixes the last of our CI issues on the main branch. Can you try rebasing against main so we can run CI again? Once CI passes I think we should be good to merge!

**Comment by sorcix** [comment]
Rebased. Also tried running `cargo fmt --all` in my branch, but it doesn't change anything.

**Comment by sorcix** [comment]
The [check wasm target](https://github.com/http-rs/http-types/runs/4097374602?check_suite_focus=true) test failed. I don't think that's related to my change, though.

**Comment by Fishrock123** [comment]
I don't know why rustup failed in that CI in november but it doesn't anymore

**Pull request merged**